proto/spacemesh/v1/smesher_types.proto

```diff
@@ -4,8 +4,14 @@ option go_package = "github.com/spacemeshos/api/release/go/spacemesh/v1";
 import "spacemesh/v1/types.proto";
 import "google/rpc/status.proto";
 
-message IsSmeshingResponse {
-    bool is_smeshing = 1;
+message SmeshingStatusResponse {
+    enum SmeshingStatus {
```

Comment:
I would change this to PosInitStatus. POS = proof of space without the time part.

Reply:
I’m not sure that using PoS instead of PoST is correct. What we’re doing here is PoST init, not PoS.

Reply:
I saw that you used this abbreviation in other places as well.

Reply:
I disagree - the time element is only added by the poet interactions after the initial space is setup. It is much simpler to call the first phase (init, space commitment) proof of space setup and everything else under the 'smeshing' umbrella term which includes the time dimension involving the poet interactions, etc... What i like to communicate to users and let them understand that what they do is first commit space from their drive and next the protocol turns it into proofs of space time which gives them eligibility in different epochs... we have lots of more work on good naming of things in this area but using 'pos' and 'smeshing' is a good start imo...

Reply:
Continued discussion: https://github.com/spacemeshos/api/pull/145#discussion_r643439432